src/Sulu/Bundle/WebsiteBundle/Resources/public/js/components/webspace/settings/analytics/main.js

```diff
@@ -0,0 +1,172 @@
+                        resultKey: 'analytics',
+                        actionCallback: this.editAnalytics.bind(this),
+                        pagination: 'infinite-scroll',
+                        viewOptions: {
+                            table: {
+                                actionIconColumn: 'title'
+                            }
+                        },
+                        matchings: [
+                            {
+                                attribute: 'title',
+                                content: this.translations.title
+                            },
+                            {
+                                attribute: 'type',
+                                content: this.translations.type
+                            },
+                            {
+                                attribute: 'domains',
+                                content: this.translations.domains,
+                                type: function(content) {
+                                    if (content === true) {
+                                        return this.translations.allDomains;
+                                    }
+
+                                    var urls = _.map(content, function(item) {
+                                        return item['url'];
+                                    });
+
+                                    return urls.join(', ');
+                                }.bind(this)
+                            }
+                        ]
+                    }
+                }
+            ]);
+        },
+
+        editAnalytics: function(id) {
+            this.sandbox.start([
+                {
+                    name: 'webspace/settings/analytics/overlay@suluwebsite',
+                    options: {
+                        el: '#webspace-analytics-form-overlay',
+                        id: id,
+                        webspaceKey: this.data.key,
+                        urls: this.data.urls,
+                        saveCallback: this.save.bind(this),
+                        translations: this.translations
+                    }
+                }
+            ])
+        },
+
+        save: function(id, data) {
+            this.sandbox.util.save(
+                this.templates.url({webspace: this.data, id: id, ids: null}), !!id ? 'PUT' : 'POST', data
+            ).then(function(response) {
+                var event = 'husky.datagrid.record.add';
+                if (!!id) {
+                    event = 'husky.datagrid.records.change';
+                }
+
+                this.sandbox.emit(event, response);
+                this.sandbox.emit('sulu.labels.success.show', this.translations.successMessage, this.translations.successLabel);
+            }.bind(this));
+        },
+
+        deleteAnalytics: function() {
+            var ids = JSON.parse(JSON.stringify($('#webspace-analytics-list').data('selected')));
```

Comment:
I'd say it would be more elegant to use the `this.sandbox.util.deepCopy` method instead